filebeat/beater/publish.go

```diff
@@ -204,6 +217,7 @@ func (p *asyncLogPublisher) collect() bool {
 		// Tell the registrar that we've successfully sent these events
 		select {
 		case <-p.done:
+			logp.Info("Shutting down - No registrar update for successfully published batch.")
```

Comment:
Comment 2 lines above says differently :-) Why don't we update the registrar here?